tests/smoke-tests/test-staking-rewards.ts

```diff
@@ -6,23 +6,307 @@ import { ApiPromise } from "@polkadot/api";
 import { expect } from "chai";
 import { describeSmokeSuite } from "../util/setup-smoke-tests";
 import { HexString } from "@polkadot/util/types";
+import { PalletParachainStakingDelegator } from "@polkadot/types/lookup";
+import { ApiDecoration } from "@polkadot/api/types";
+import Bottleneck from "bottleneck";
+import { AccountId20 } from "@polkadot/types/interfaces";
+import { FIVE_MINS, ONE_HOURS, TWO_HOURS } from "../util/constants";
 import { Perbill, Percent } from "../util/common";
 const debug = require("debug")("smoke:staking");
 
-if (!process.env.SKIP_BLOCK_CONSISTENCY_TESTS) {
-  describeSmokeSuite(`Verify staking rewards`, function (context) {
-    it("rewards are given as expected", async function () {
-      this.timeout(500000);
-      if (context.polkadotApi.consts.system.version.specVersion.toNumber() < 2000) {
-        this.skip();
+describeSmokeSuite(`When verifying ParachainStaking rewards...`, function (context) {
```

Comment:
Perhaps it makes more sense to move this test to its own file, since it's pretty much independent of rewards.

Reply:
its going to be integrated into rewards eventually, so makes sense for it all to be in same place. These tests will verify the snapshot, and the existing tests will use that snapshot against payouts made